src/nethsec/users/__init__.py

```diff
@@ -262,10 +263,12 @@ def add_ldap_database(uci, name, uri, schema, base_dn, user_dn, user_attr, user_
   if bind_dn and bind_password:
       uci.set('users', name, 'bind_dn', bind_dn)
       uci.set('users', name, 'bind_password', bind_password)
+  if user_bind_dn is not None:
```

Comment:
```suggestion
  if user_bind_dn:
```
According to https://github.com/NethServer/nethsecurity-ui/pull/303, UI puts `user_bind_dn=""` if the user doesn't set a custom user bind DN in the form

Reply:
Actually the behavior is the same: the UI sets an empty string and then the field gets deleted by UCI.
I'm still going to merge the suggestions just to have a more uniform code.